src/Yoti.Auth/Profile/YotiProfile.cs

```diff
@@ -182,6 +182,19 @@ public YotiAttribute<List<Image>> DocumentImages
             }
         }
 
+        /// <summary>
+        /// IdentityProfileReport represents the JSON object containing identity assertion and the
+        /// verification report. This will be null if not provided by Yoti.
+        /// </summary>
+        public YotiAttribute<Dictionary<string, JToken>> IdentityProfileReport
```

Comment:
Ed. Just wondering, in the same vein as discussion on the Java side... might we perhaps be better off here using `YotiAttribute<dynamic> `
as opposed 
`YotiAttribute<Dictionary<string, JToken>>`
...as although the latter should handle the current json schema fine - it will (I believe) break if the IdentityProfileReport was ever changed to an array (as opposed to an object) at it's root. I think this is probably unlikely, but worth mentioning perhaps.

Reply:
@pgreci I know it would be a breaking change, but do you think we ever might change the IdentityProfileReport to have an array, rather than an object with string keys? 
My thinking would be that if that were the case, we'd introduce `identity_profile_reports` (plural), so as not to be a breaking change. But just wanted to confirm.

Reply:
If I understand well, isn't it the same case as for the `document_images` which could now appear several time and for which we need to introduce a way of managing all its occurrence by introducing something like `getAttributeByNameAndId()`.... if so, the same principle could be applied to the attribute `identity_profile_report` and in case there are several of them, we'd be able to find the right one by specifying an extra id  ?

Reply:
Yep, think we just get the right one by ID, I don't think we need `name` as well, assuming the IDs are unique? So `getAttributeById`?